### ISSUE TITLE
neotree strange gray background with astrovim

### ISSUE THREAD

**Issue by lunemec** [navarasu/onedark.nvim#193]
Hello, I'm using astrovim, and for some reason the neotree with this theme changes color after re-opening the neotree.

When I initially open empty nvim, and open tree, it has correct color on the background. But on any opening of file in the neotree or any other action that causes the neotree to reopen some folders, it changes the bg to this strange gray.

I don't know why, or if its bug or feature, nor in which plugin. Any ideas?

![before](https://github.com/navarasu/onedark.nvim/assets/1670632/ad14138d-676c-482e-b2ca-cc2af695d918)
![after](https://github.com/navarasu/onedark.nvim/assets/1670632/c6c1ca7b-4a81-4b6b-a404-4cf8118211c5)

**Comment by nvergottini** [comment]
I'm seeing the same gray background with Neotree and LazyVim except that the gray background is there when I first open it.

**Comment by DennisLoska** [comment]
yeah super annoying. When I use another color scheme like tokyonight it works just fine.

**Comment by deanqx** [comment]
it seems to be the bg3 color
